partner_sale_pivot/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,4 @@
+- [Tecnativa](https://www.tecnativa.com):
+  - Ernesto Tejeda
+  - Pedro M. Baeza
+  - Sergio Teruel
```

Comment:
@Wodran14 FYI, when doing a migration, you can add yourself to contributors :smiley: